www/app/manage.php

```diff
@@ -1,7 +1,7 @@
 <?php
 	// Logic for ALL Management Pages
 	include_once 'global.php';
-	if (!isset($_SESSION['email'])) {
+	if (!isLoggedIn()) {
```

Comment:
moved this session check to a method.  If down the road implementation changes it can be updated in one place.